llvm/lib/IR/RuntimeLibcalls.cpp

```diff
@@ -277,5 +277,10 @@ void RuntimeLibcallsInfo::initLibcalls(const Triple &TT) {
       setLibcallName(RTLIB::MULO_I64, nullptr);
     }
     setLibcallName(RTLIB::MULO_I128, nullptr);
+  } else {
```

Comment:
This particular change LGTM, but @nikic, can you explain where the `!TT.isWasm` above came from? It looks like you added it in dad9e4a.

Reply:
IIRC to preserve existing behavior and because these libcalls were listed in RuntimeLibcallSignatureTable.

I don't think wasm is typically used with libgcc? Is that supported?

Reply:
AFAIK there are no uses of libgcc with wasm.